tgui/packages/tgui/interfaces/Pod/Parts/DNAPart.tsx

```diff
@@ -0,0 +1,35 @@
+import { useBackend } from '../../../backend';
+import { Box, Button, Icon, Stack } from '../../../components';
+type Data = {
```

Comment:
```suggestion
type Props = {
```
we use Data for backend data as convention. This is just props